tools/maven/suppressions.xml

```diff
@@ -43,7 +43,11 @@ under the License.
 		<suppress
 			files="FlinkKinesisProducer.java|FlinkKinesisProducerTest.java"
 			checks="IllegalImport"/>
-		<!-- Classes copied from Hadoop -->
+		<!-- Kinesis EFO consumer required to handle Netty ReadTimeoutException -->
+		<suppress 
+			files="FanOutRecordPublisherTest.java|FanOutShardSubscriber.java|FanOutShardSubscriberTest.java" 
+			checks="IllegalImport"/>
```

Comment:
Is this due to the fact that these classes are using non-shaded netty imports?

Reply:
Yes non-shaded at check time. The Netty deps are shaded in the final build.